feature/user_profile/presentation/src/test/java/com/sanaa/presentation/myAccount/MyRatingScreenViewModelTest.kt

```diff
@@ -0,0 +1,197 @@
+package com.sanaa.presentation.myAccount
+
+import app.cash.turbine.test
+import com.google.common.truth.Truth
+import com.google.common.truth.Truth.assertThat
+import com.sanaa.identity.dataSoruce.local.dataStore.PreferencesManager
+import com.sanaa.presentation.screen.myRating.MediaTypeUi
+import com.sanaa.presentation.screen.myRating.MyRatingScreenEffect
+import com.sanaa.presentation.screen.myRating.MyRatingScreenViewModel
+import com.sanaa.presentation.screen.myRating.MyRatingTab
+import entity.Movie
+import entity.TvSeries
+import exceptions.NoNetworkException
+import io.mockk.coEvery
+import io.mockk.mockk
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.ExperimentalCoroutinesApi
+import kotlinx.coroutines.flow.flowOf
+import kotlinx.coroutines.test.StandardTestDispatcher
+import kotlinx.coroutines.test.advanceUntilIdle
+import kotlinx.coroutines.test.runTest
+import kotlinx.coroutines.test.setMain
+import kotlinx.coroutines.withTimeout
+import kotlinx.datetime.LocalDate
+import org.junit.jupiter.api.BeforeEach
+import org.junit.jupiter.api.Test
+import usecase.ManageMovieUseCase
+import usecase.ManageTvSeriesUseCase
+
+@OptIn(ExperimentalCoroutinesApi::class)
+class MyRatingScreenViewModelTest {
+    private val manageMovieUseCase: ManageMovieUseCase = mockk(relaxed = true)
+    private val manageTvSeriesUseCase: ManageTvSeriesUseCase = mockk(relaxed = true)
+    private val preferencesManager: PreferencesManager = mockk(relaxed = true)
+    private lateinit var viewModel: MyRatingScreenViewModel
+    private val testDispatcher = StandardTestDispatcher()
+
+    @BeforeEach
+    fun setUp() {
+        Dispatchers.setMain(testDispatcher)
+        coEvery { preferencesManager.accountId } returns flowOf(123L)
+        coEvery { preferencesManager.sessionId } returns flowOf("test_session")
+    }
+
+    @Test
+    fun `loadRatedMedia sets isNoInternetConnection to true on movie network error`() = runTest {
```

Comment:
it's ok, but use (should when) it's good practice 
```suggestion
    fun `loadRatedMedia should sets isNoInternetConnection to true when there is a network error`() = runTest {
```